rust/lance/src/dataset/transaction.rs

```diff
@@ -199,6 +199,25 @@ pub enum Operation {
     },
 }
 
+impl std::fmt::Display for Operation {
```

Comment:
nit: we could get this for free from [strum_macros](https://docs.rs/strum_macros/latest/strum_macros/derive.Display.html) but would need to add an explicit dependency